doc/source/release-notes/Percona-Server-5.5.46-37.5.rst

```diff
@@ -0,0 +1,41 @@
+.. rn:: 5.5.46-37.6
+
+==============================
+ |Percona Server| 5.5.46-37.5
+==============================
+
+Percona is glad to announce the release of |Percona Server| 5.5.46-37.5 on November 5th, 2015. Downloads are available `here <http://www.percona.com/downloads/Percona-Server-5.5/Percona-Server-5.5.46-37.5/>`_ and from the :doc:`Percona Software Repositories </installation>`.
+
+Based on `MySQL 5.5.46 <http://dev.mysql.com/doc/relnotes/mysql/5.5/en/news-5-5-46.html>`_, including all the bug fixes in it, |Percona Server| 5.5.46-37.5 is now the current stable release in the 5.5 series. All of |Percona|'s software is open-source and free, all the details of the release can be found in the `5.5.46-37.5 milestone at Launchpad <https://launchpad.net/percona-server/+milestone/5.5.46-37.5>`_. 
+
+New Features
+============
+
+ |Percona Server| is now available for *Ubuntu* 15.10 (Wily).
+
+Bugs Fixed
+==========
+
+ Running ``PURGE CHANGED_PAGE_BITMAPS BEFORE 0`` could cause server to crash. Bug fixed :bug:`1368530`.
```

Comment:
Manipulating innodb_track_redo_log_now variable dynamically would crash the server if it was started without innodb_track_changed_pages enabled. Note that this variable is available on debug builds only.